js/src/admin/addSettingsModal.js

```diff
@@ -1,38 +1,105 @@
-import { settings } from '@fof-components';
-import Badge from 'flarum/components/Badge';
+import { settings } from "@fof-components";
+import Badge from "flarum/components/Badge";
 
 const {
     SettingsModal,
-    items: { BooleanItem, StringItem, SelectItem },
+    items: { BooleanItem, StringItem, SelectItem }
 } = settings;
 
+function visualiseTags() {
+    let secondaryTags = {};
+
+    const tags = app.store
+        .all("tags")
+        .sort((a, b) => a.data.attributes.position - b.data.attributes.position)
+        .reduce((options, object) => {
+            if (/*object.isChild() ||*/ object.position() === null) {
```

Comment:
As `object.isChild()` is no longer required and commented out, let's remove it to keep the code nice and clean ;)

Reply:
Poof! It's gone! :)